MonoDevelop.MSBuild/Schema/BuiltInSchemaId.cs

```diff
@@ -28,5 +28,6 @@ enum BuiltInSchemaId
 	ProjectSystemMps,
 	ProjectSystemCps,
 	GenerateAssemblyInfo,
-	ValidatePackage
+	ValidatePackage,
+	ILCompiler
```

Comment:
Named after [this project](https://github.com/dotnet/runtime/tree/99dd60d8886155bb60768c8ef5a5a3225ad95ba4/src/coreclr/tools/aot/ILCompiler) rather than "NativeAot", but happy to change if that's preferred.